src/lib/apiErrorHandler.js

```diff
@@ -144,3 +121,113 @@ export default async function handleErrors(req, res, error, logContext) {
     handleUnexpectedErrorInCatchBlock(req, res, e, logContext);
   }
 }
+
+/**
+ * Logs a worker error and stores it in the database.
+ * @param {Object} job - The job object.
+ * @param {string} operation - The operation name.
+ * @param {Error} error - The error object.
+ * @param {Object} logContext - The logging context.
+ * @returns {Promise<number|null>} - The ID of the stored request error, or null if storing failed.
+ */
+const logWorkerError = async (job, operation, error, logContext) => {
+  if (
+    operation !== 'SequelizeError'
+    && process.env.SUPPRESS_ERROR_LOGGING
+    && process.env.SUPPRESS_ERROR_LOGGING.toLowerCase() === 'true'
+  ) {
+    return 0;
+  }
+
+  try {
+    const responseBody = typeof error === 'object' && error !== null
+      ? { ...error, errorStack: error?.stack }
```

Comment:
Wouldn't ...error also contain error.stack. Why do we need to put it in the property errorStack?

Reply:
just matching what the original code had